tests/python/distributed/test_data_parallel.py

```diff
@@ -77,18 +79,28 @@ def test_data_parallel():
 )
 @with_seed(0)
 def test_zero_opt_1():
-    dctx = dist.get_context()
-    dctx.enable_data_parallel = True
-    dctx.zero_opt_level = 1
-    device = f"cuda({dctx.local_rank})"
+    dcfg = dist.get_config()
+    dcfg.enable_data_parallel = True
+    dcfg.zero_opt_level = 1
+    comm = dist.get_communicator()
+    device = f"cuda({comm.local_rank})"
 
     run_model(device)
 
-    dctx.enable_data_parallel = False
-    dctx.zero_opt_level = 0
+    dcfg.enable_data_parallel = False
+    dcfg.zero_opt_level = 0
 
 
 if __name__ == "__main__":
+    if os.environ.get("RAF_FILE_STORE_PATH", None):
+        dist.set_default_communicator("void")
```

Comment:
I am a little bit confused here. If I understand correctly, the default communicator should be `MPI` and we do not need to set `RAF_FILE_STORE_PATH`for MPI. The test is for `void` communicator?